buffer/src/test/java/io/netty/buffer/AbstractByteBufTest.java

```diff
@@ -1846,7 +1850,7 @@ public void run() {
                                 return;
                             }
                         }
-                        Assert.assertArrayEquals(bytes, out.toByteArray());
+                        assertArrayEquals(bytes, out.toByteArray());
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this assertion. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS2186)